scripts/checkServicesTasks.js

```diff
@@ -0,0 +1,122 @@
+const RedisSMQ = require('rsmq');
+const Redis = require('redis');
+
+const { tenant, redisUrl, read, deleteAll, service, deleteOne } = require('yargs')
+  .option('tenant', {
+    alias: 't',
+    type: 'string',
+    describe: 'Tenant to check',
+  })
+  .option('redisUrl', {
+    alias: 'u',
+    type: 'string',
+    describe: 'Redis url',
+    default: 'redis://localhost:6379',
+  })
+  .option('service', {
+    alias: 's',
+    type: 'string',
+    describe: 'Service queue',
+    choices: ['information_extraction', 'segmentation'],
+  })
+  .option('read', {
+    alias: 'r',
+    type: 'boolean',
+    describe: 'Read the queue',
+    default: true,
+  })
+  .option('deleteAll', {
+    alias: 'D',
+    type: 'boolean',
+    describe: 'Delete all messages from the queue',
+    default: false,
+  })
+  .option('deleteOne', {
+    alias: 'd',
+    type: 'string',
+    describe: 'ID of the message to be deleted',
+  })
+
+  .demandOption(['service'], '\n\n').argv;
+
+const run = async () => {
+  const redisClient = await Redis.createClient(redisUrl);
```

Comment:
While there are these in other places of the code as well, my typescript tells me that these two awaits may be not necessary.

Reply:
right, done